sources/packages/forms/Dockerfile

```diff
@@ -4,10 +4,13 @@
 
 # Use Node image, maintained by Docker:
 # hub.docker.com/r/_/node/
-FROM docker-remote.artifacts.developer.gov.bc.ca/node:12.2.0-alpine
+FROM docker.io/library/node:lts-alpine3.10
```

Comment:
I believe that this is the build that will run on OpenShift, right?
We should not skip the `artifacts`. The image must come from there.